src/lib/parser_impl.c

```diff
@@ -589,8 +657,32 @@ parser_error_t _getCommissionBoundStepAtIndex(parser_context_t *c, parser_tx_t *
       CHECK_CBOR_ERR(cbor_value_advance(&boundsContainer));
     }
 
-    CHECK_CBOR_ERR(_readBound(&boundsContainer, &v->oasis_tx.body.stakingAmendCommissionSchedule.bound));
+    CHECK_CBOR_ERR(_readBound(&boundsContainer, &v->oasis.tx.body.stakingAmendCommissionSchedule.bound));
 
     return parser_ok;
 
 }
+
+parser_error_t _getNodesIdAtIndex(parser_context_t *c, parser_tx_t *v, uint8_t index) {
+    CborValue it;
+    CHECK_CBOR_ERR(cbor_parser_init(c->buffer,
+                                    c->bufferLen,
+                                    c->offset,
+                                    &v->parser,
+                                    &it));
+
+    if (cbor_value_at_end(&it)) {
+        return parser_unexpected_buffer_end;
+    }
+
+    CborValue nodesContainer;
+    CHECK_CBOR_ERR(_getBoundsContainer(&it, &nodesContainer));
```

Comment:
Is this correct? It seems to be the wrong function to call here.

Reply:
yeah that was a mistake. It should be resolved now.